testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpServerTest.groovy

```diff
@@ -282,257 +176,137 @@ abstract class HttpServerTest<SERVER> extends InstrumentationSpecification imple
     }
   }
 
-  def "test success with #count requests"() {
-    setup:
-    def request = request(SUCCESS, method)
-    List<AggregatedHttpResponse> responses = (1..count).collect {
-      return client.execute(request).aggregate().join()
+  @Shared
+  def junitTest = new AbstractHttpServerTest<Void>() {
+    @Override
+    protected Void setupServer() {
+      return null
     }
 
-    expect:
-    responses.each { response ->
-      assert response.status().code() == SUCCESS.status
-      assert response.contentUtf8() == SUCCESS.body
+    @Override
+    protected void stopServer(Void o) {
     }
 
-    and:
-    assertTheTraces(count, null, null, method, SUCCESS, null, responses[0])
+    @Override
+    protected void configure(HttpServerTestOptions options) {
+      options.expectedServerSpanNameMapper = { endpoint, method ->
+        HttpServerTest.this.expectedServerSpanName(endpoint, method)
+      }
+      options.expectedHttpRoute = { endpoint ->
+        HttpServerTest.this.expectedHttpRoute(endpoint)
+      }
+      options.contextPath = getContextPath()
+      options.httpAttributes = { endpoint ->
+        HttpServerTest.this.httpAttributes(endpoint)
+      }
+      options.expectedExceptionClass = expectedExceptionClass()
+      options.hasExceptionOnServerSpan = { endpoint ->
+        HttpServerTest.this.hasExceptionOnServerSpan(endpoint)
+      }
+
+      options.testRedirect = testRedirect()
+      options.testError = testError()
+      options.testErrorBody = testErrorBody()
+      options.testException = testException()
+      options.testNotFound = testNotFound()
+      options.testPathParam = testPathParam()
+      options.testCaptureHttpHeaders = testCapturedHttpHeaders()
+      options.testCaptureRequestParameters = testCapturedRequestParameters()
+    }
+
+    @Override
+    void assertTheTraces(
+        int size,
+        String traceId,
+        String parentId,
+        String method,
+        ServerEndpoint endpoint,
+        String errorMessage,
+        AggregatedHttpResponse response) {
+      HttpServerTest.this.assertTheTraces(size, traceId, parentId, method, endpoint, errorMessage, response)
+    }
+
+    void assertHighConcurrency(int count) {
+      HttpServerTest.this.assertHighConcurrency(count)
+    }
```

Comment:
can we get rid of the local groovy implementations of these two methods and only use the java implementations of them?
```suggestion
```